test/14_release-pinning.ts

```diff
@@ -343,7 +363,7 @@ describe(`Tracking latest release`, () => {
 			});
 			app4ReleaseId = app4Release.id;
 			const app5Release = await addReleaseToApp(admin, {
-				belongs_to__application: application3Id,
+				belongs_to__application: applicationToDelete.id,
```

Comment:
I would prefer it if we didn't touch pre-existing releases, since this could influence whether existing test cases actually test what they should any more.
Let's just create new releases.